lib/containers/utils.pm

```diff
@@ -41,20 +41,56 @@ sub test_seccomp {
     }
 }
 
-sub basic_container_tests {
-    my %args    = @_;
-    my $runtime = $args{runtime};
-    die "You must define the runtime!" unless $runtime;
+sub check_docker_firewall {
+    my $container_name = 'sut_container';
+    validate_script_output('ip a s docker0', sub { /state DOWN/ });
```

Comment:
Why do you check that the `docker0` is `down`? There might be other containers running.
If you are sure that when no containers are running the `docker0` is `down` then please test that.

Reply:
`check_docker_firewall` is executed after `install_docker_when_needed`. in this case the most possible is that no containers are running. But i guess it adds an extra net if we provide a check. wdyt?

Reply:
i added a line for checking